### PR TITLE
Changes to Claims school user locales

### PR DESCRIPTION
**Pull request by JamieCleare2525**
## Context

Locales for Claims - Schools - Users pages currently do not match the prototype

## Changes proposed in this pull request

screencapture-claims-localhost-3000-support-schools-ffff4753-e523-4e3b-a27d-9ba6f2dc1308-users-2024-01-17-15_57_52

screencapture-claims-localhost-3000-support-schools-ffff4753-e523-4e3b-a27d-9ba6f2dc1308-users-new-2024-01-17-15_58_03

## Guidance to review

Text on the Claims - Organisations - Users pages should now match the prototype

### CODE REVIEW

File: config/locales/claims/en/support/schools/users.yml [hunk 1]
```diff
@@ -18,9 +18,9 @@ en:
                 first_name: First name
                 last_name: Last name
                 email: Email
```
Comment (CatalinVoineag) [inline config/locales/claims/en/support/schools/users.yml:20]: I think on the Prototype this is Email address 
Screenshot from 2024-01-17 16-13-11

**Review by CatalinVoineag — APPROVED** [review]
Looks good, only 1 little issue I found

File: config/locales/claims/en/support/schools/users.yml (hunk 1 above)
Comment (Nitemaeric) [reply config/locales/claims/en/support/schools/users.yml:20]: I've started on some work that uses `User.human_attribute_name(:email)` => `Email address` for these labels. FYI :)

Comment (JamieCleare2525) [reply config/locales/claims/en/support/schools/users.yml:20]: Good catch @CatalinVoineag . Resolved in [f5e3dbb](https://github.com/DFE-Digital/itt-mentor-services/pull/132/commits/f5e3dbbbc8c592c86d73fdf19629fc3d0a5aaf11)

**Pull request merged**